lib/internal/debugger/inspect_client.js

```diff
@@ -193,7 +193,17 @@ class Client extends EventEmitter {
       if (payloadBuffer === null || payloadBuffer.length === 0) break;
 
       const payloadStr = payloadBuffer.toString();
-      debuglog('< %s', payloadStr);
+      if (payloadStr.startsWith('{"method":"Debugger.scriptParsed"') &&
+          payloadStr.includes('"url":"node:')) {
+        // Do not log Debugger.scriptParsed events for internal scripts
+        // because the payloads can be huge.
```

Comment:
```suggestion
        // because the payloads can be huge.
        debuglog('< Debugger.scriptParsed: ', payloadStr.substring(100), '...');
```

Reply:
This is more or less temporary for testing purposes, I am going to revert most of the last two commits.